mappings/net/minecraft/world/biome/BiomeGenerator.mapping

```diff
@@ -1 +1,13 @@
 CLASS none/ajk net/minecraft/world/biome/BiomeGenerator
+	FIELD a isDecorating Z
```

Comment:
As per my suggestion / addition to [the guidelines](https://docs.google.com/document/d/15fHL-WgK0uMPAy-WJbQtxrfOVLJNlfasjsnt_wruOXA/edit), it should be `decorating`.